File: app/src/main/kotlin/com/moviereel/di/components/ServiceComponent.java
package com.moviereel.di.components;

import com.moviereel.di.PerService;
import com.moviereel.di.modules.ServiceModule;

import dagger.Component;

/**
 * @author lusinabrian on 27/03/17
 */

@PerService
@Component(dependencies = ApplicationComponent.class, modules = ServiceModule.class)
public interface ServiceComponent {

    // TODO: 27/03/17 inject services here
    // void inject();
}

File: app/src/main/kotlin/com/moviereel/di/components/ServiceComponent.kt
package com.moviereel.di.components

import com.moviereel.di.PerService
import com.moviereel.di.modules.ServiceModule

import dagger.Component

/**
 * @author lusinabrian on 27/03/17
 */

@PerService
@Component(dependencies = arrayOf(ApplicationComponent::class), modules = arrayOf(ServiceModule::class))
interface ServiceComponent// TODO: 27/03/17 inject services here
// void inject();
